src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -23,6 +23,7 @@ public class FunctionMetadataManager : IFunctionMetadataManager
         private readonly IFunctionMetadataProvider _functionMetadataProvider;
         private bool _isHttpWorker;
         private bool _servicesReset = false;
+        private bool _jobHostServicesInitialized = true;
```

Comment:
this should start at false? and then set to true in HostInitializing the call back right?

Reply:
Forgot to reverse booleans with name change haha. Fixed.